### PR TITLE
Adding type and abis

### PR DESCRIPTION
**Pull request by bengriffin1**

### CODE REVIEW

File: src/modules/utils/index.ts [hunk 1]
```diff
@@ -12,4 +15,62 @@ export class UtilsModule extends BaseModule {
 
     return header.substring(7);
   }
+
+  // Token Gating function validates user ownership of wallet + NFT
+  public async validateTokenOwnership(
+    didToken: string,
+    contractAddress: string,
+    contractType: 'ERC721' | 'ERC1155',
+    web3: Web3,
```
Comment (Ethella) [inline src/modules/utils/index.ts:24]: Curious, why web3 needs to be a param in this case?

Comment (Ethella) [reply src/modules/utils/index.ts:24]: Just a thought, maybe web3 can be a peer dependency, which can keep our admin sdk light-weight and thin

Comment (bengriffin1) [reply src/modules/utils/index.ts:24]: I made it a param to make the mocks more straightforward in the unit tests, and also to allow the users to pass in web3 instances that may have custom options.

I'm not familiar with peer dependencies, looking into it

Comment (bengriffin1) [reply src/modules/utils/index.ts:25]: moved into peer dependencies @Ethella

**Review by Ethella — APPROVED** [review]

**Comment by Ethella** [comment]
:rocket: PR was released in [`v2.2.0`](https://github.com/magiclabs/magic-admin-js/releases/tag/v2.2.0) :rocket:

**Pull request merged**